teamchallenge.html

```diff
@@ -55,18 +57,31 @@ <h1>Team Challenge</h1>
 
     <h2>The Task</h2>
 
-    <p>Your team will be responsible for creating a Command Line Interface version of the game rock paper scissors.</p>
-
-    <p>A user should be able to play against the computer. Each “game” should be scored best 2 out of 3.</p>
-
-    <p>Instead of repl.it you will need to use <a href="https://www.jdoodle.com/execute-nodejs-online" target="_blank">jdoodle</a>. Jdoodle will allow you to add interactivity to your game.</p>
+    <p>Your team will be responsible for creating a version of the game Rock, Paper, Scissors.</p>
 
-    <p>You will need to use a JS module called readline to get user input. <a href="https://nodejs.org/api/readline.html" target="_blank">Readline documentation</a>.</p>
+    <p>Core gameplay:</p>
+    <ul>
+      <li>A prompt appears on the screen asking the user to select Rock, Paper, or Scissors</li>
+      <li>When the user clicks "OK", their throw is logged to the console</li>
+      <li>The computer randomly selects Rock, Paper, or Scissors, then the computer's throw is logged to the console</li>
+      <li>Your program looks at the two throws, and logs a message to the console that says if the player won, lost or if the game was a tie</li>
+    </ul>
 
-    <p>Here is an example of a CLI game: <a href="https://drive.google.com/file/d/14IXuxOBU-Fhe5XyNyd8amkDHkMoqJc0m/view?usp=sharing" target="_blank">Rock Paper Scissors</a>. Everything you see is taking place in the console. In this example the input “node rps.js” is the equivalent of hitting “run”.</p>
+    <p>Advanced Features:</p>
+    <ul>
+      <li>Have a prompt come up that lets the player play the game again</li>
+      <li>If both the user and the computer pick the same thing, have the user/computer keep entering new throws until there is a winner or loser</li>
+      <li>Make a game a best of 3 instead of a single throw</li>
+    </ul>
 
+    <p>Important resources:</p>
+    <ul>
+      <li>In order to get prompts to work, you will need to use a repl at <a href = "https://repl.it/languages/html" target="_blank">this link here</a></li>
+      <li>Click on "script.js" on the left to start writing javascript, and click on "console" on the right to see the console</li>
+      <li>Use <b>prompt("Your prompt here")</b> to make an alert with a text field appear on the screen.  More information can be found at the <a href = "https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt" target = "_blank">documentation here</a></li>
```

Comment:
@benstone1 does this work? i found that `repl.it` has this error: `ReferenceError: prompt is not defined`